README.md

```diff
@@ -7,36 +7,41 @@ Rami Matar (rhm2142), Mustafa Eyceoz (me2680), Justin Lee (jjl2245)
 
 ## Project Summary
 
-Diffusion models are very effective learners of data distributions that learn to denoise Gaussian-blurred images and reverse the steps of the diffusion process. The diffusion process is essentially a noising process in which Gaussian noise is applied to an input image repeatedly for T steps. Initially [2], diffusion models trained a sequence of T autoencoders to learn the denoising process in the pixel space. A much more computationally efficient variant of the diffusion model, known as the Latent Diffusion Model (LDM) works by performing the diffusion process within the latent space of a pre-trained autoencoder [1]. These models have shown very competitive results on many image synthesis tasks, and can further allow for learning many conditional distributions simultaneously without much change to training.
+The goal of this project is to enable accessible, beautiful art generation (and conditional art generation) via latent diffusion models. Through a combination of a VQGAN, Diffusion, a UNet model with self-attention layers, and additional training/transfer-learning techniques, our system can convert any generated small noise pattern (32x32x4) into a hand-painted masterpiece. While small, compact, and fast, the art quality is still capable of making one stare and ponder. Additionally, users may soon be able to input captions as inspiration for their generated works.
```

Comment:
This project explores latent diffusion models introduced in the paper  "High-Resolution Image Synthesis with Latent Diffusion Models" by Robin Rombach et. al. (2021). Diffusion models are  generative models that have achieved state of the art results in many image processing tasks. Particularly, we explore the generative power of these models as well as their ability to learn conditional distributions (such as text). Then continue the rest after